docs/integrations/azure/discovery.md

```diff
@@ -67,7 +67,7 @@ The parameters available are:
 
 - **`host:`** _(optional)_ Leave empty for Cloud hosted, otherwise set to your self-hosted instance host.
 - **`organization:`** Your Organization slug (or Collection for on-premise users). Required.
-- **`project:`** _(optional)_ Your project slug. Wildcards are supported as show on the examples above. If not set, all projects will be searched. For a project name containing spaces, use both single and double quotes as in `project: '"My Project Name"'`.
+- **`project:`** _(required)_ Your project slug. Wildcards are supported as show on the examples above. If set '*' will search all projects. For a project name containing spaces, use both single and double quotes as in `project: '"My Project Name"'`.
```

Comment:
Can we add an "n" to the end of "show"? So it should read: "Wildcards are supported as shown on the examples above."

Also, this line "If set '*' will search all projects." would be better without the "If". How about: "Using a wildcard '*" will search all projects."

Reply:
@awanlin i fixed :)